rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -37,6 +40,28 @@ def preprocess_pdf_content(content: str):
     return content
 
 
+def _load_pdf_using_pypdf(file_path: str) -> List[Document]:
```

Comment:
Severity: Potential Issue - The function name '_load_pdf_using_pypdf' could be more descriptive. Consider renaming it to 'load_pdf_with_pypdf' for clarity.